### ISSUE TITLE
[PRE REVIEW]: portalcasting: Supporting automated forecasting of rodent populations

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#3027]
**Submitting author:** @gmyenni (<a href="http://orcid.org/0000-0001-6969-1848">Glenda Yenni</a>)
**Repository:** <a href="http://github.com/weecology/portalcasting" target ="_blank">http://github.com/weecology/portalcasting</a>
**Version:** v0.20.0
**Editor:** @marcosvital
**Reviewers:** @ViniciusBRodrigues , @FlukeAndFeather 
**Managing EiC:** Arfon Smith

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**Author instructions**

Thanks for submitting your paper to JOSS @gmyenni. **Currently, there isn't an JOSS editor assigned** to your paper.

The author's suggestion for the handling editor is @marcosvital.

@gmyenni if you have any suggestions for potential reviewers then please mention them here in this thread (without tagging them with an @). In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission (please start at the bottom of the list).

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.88  T=0.64 s (106.2 files/s, 22912.1 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
R                               50           1236           4858           6172
Markdown                         7            225              0            687
YAML                             3              6              0            294
Rmd                              5            287            459            247
TeX                              1              8              0            114
JSON                             1              0              0             60
Dockerfile                       1              5              3              6
-------------------------------------------------------------------------------
SUM:                            68           1767           5320           7580
-------------------------------------------------------------------------------


Statistical information for the repository 'c1e5c30e989adb257bf4f818' was
gathered on 2021/02/10.
The following historical commit information, by author, was found:

Author                     Commits    Insertions      Deletions    % of changes
DAPPERstats                      2             6              6          100.00

Below are the number of rows from each author that have survived and are still
intact in the current revision:

Author                     Rows      Stability          Age       % in comments
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.21105/joss.01098 is OK
- 10.1890/08-1222.1 is OK
- 10.1890/15-2115.1 is OK
- 10.1111/2041-210X.13104 is OK
- 10.1101/332783 is OK
- 10.5281/zenodo.1475410 is OK

MISSING DOIs

- 10.1126/science.196.4292.880 may be a valid DOI for title: Competition between seed-eating rodents and ants in desert ecosystems

INVALID DOIs

- None
```

**Comment by whedon** [comment]
PDF failed to compile for issue #3027 with the following error: 

 /app/vendor/bundle/ruby/2.6.0/bundler/gems/whedon-92346a0773a4/lib/whedon/author.rb:72:in `block in build_affiliation_string': Problem with affiliations for Juniper Simonis, perhaps the affiliations index need quoting? (RuntimeError)
	from /app/vendor/bundle/ruby/2.6.0/bundler/gems/whedon-92346a0773a4/lib/whedon/author.rb:71:in `each'
	from /app/vendor/bundle/ruby/2.6.0/bundler/gems/whedon-92346a0773a4/lib/whedon/author.rb:71:in `build_affiliation_string'
	from /app/vendor/bundle/ruby/2.6.0/bundler/gems/whedon-92346a0773a4/lib/whedon/author.rb:17:in `initialize'
	from /app/vendor/bundle/ruby/2.6.0/bundler/gems/whedon-92346a0773a4/lib/whedon.rb:205:in `new'
	from /app/vendor/bundle/ruby/2.6.0/bundler/gems/whedon-92346a0773a4/lib/whedon.rb:205:in `block in parse_authors'
	from /app/vendor/bundle/ruby/2.6.0/bundler/gems/whedon-92346a0773a4/lib/whedon.rb:202:in `each'
	from /app/vendor/bundle/ruby/2.6.0/bundler/gems/whedon-92346a0773a4/lib/whedon.rb:202:in `parse_authors'
	from /app/vendor/bundle/ruby/2.6.0/bundler/gems/whedon-92346a0773a4/lib/whedon.rb:93:in `initialize'
	from /app/vendor/bundle/ruby/2.6.0/bundler/gems/whedon-92346a0773a4/lib/whedon/processor.rb:38:in `new'
	from /app/vendor/bundle/ruby/2.6.0/bundler/gems/whedon-92346a0773a4/lib/whedon/processor.rb:38:in `set_paper'
	from /app/vendor/bundle/ruby/2.6.0/bundler/gems/whedon-92346a0773a4/bin/whedon:58:in `prepare'
	from /app/vendor/bundle/ruby/2.6.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
	from /app/vendor/bundle/ruby/2.6.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
	from /app/vendor/bundle/ruby/2.6.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
	from /app/vendor/bundle/ruby/2.6.0/gems/thor-0.20.3/lib/thor/base.rb:466:in `start'
	from /app/vendor/bundle/ruby/2.6.0/bundler/gems/whedon-92346a0773a4/bin/whedon:131:in `<top (required)>'
	from /app/vendor/bundle/ruby/2.6.0/bin/whedon:23:in `load'
	from /app/vendor/bundle/ruby/2.6.0/bin/whedon:23:in `<main>'

**Comment by arfon** [comment]
@whedon invite @marcosvital as editor

**Comment by whedon** [comment]
@marcosvital has been invited to edit this submission.

**Comment by arfon** [comment]
:wave: @marcosvital - would you be able to edit this submission for JOSS?

**Comment by marcosvital** [comment]
Hi, @arfon, I can handle this submission - sorry about the delayed reply.

**Comment by marcosvital** [comment]
@whedon assign @marcosvital as editor

**Comment by whedon** [comment]
OK, the editor is @marcosvital

**Comment by marcosvital** [comment]
Hi,  @gmyenni, I will be the editor of your submission.

Please take a look at the errors compiled above regarding the pdf generation, and check if you can correct them. If you make any changes and want to try again, you can use `@whedon generate pdf` to have whedon try to create a new version (or just let me know and I'll do it).

At the same time, we can start assigning reviewers for you submission. We usually ask the authors for suggestions, so let me know if you have anyone in mind that could do this - you can paste the github profile of anyone you think that could do it (but please don't tag anyone yet).
You can also check [this list of people](https://bit.ly/joss-reviewers) that have agreed to review for JOSS, and see if any of them may be suitable to review this submission.

**Comment by gmyenni** [comment]
Hi @marcosvital Thanks! 

What about pboesu, jhollist, juanvillada, giskid, ViniciusBRodrigues, trashbirdecology, or ajpelu?

**Comment by gmyenni** [comment]
I'll work on fixing the errors. Sorry if it takes a few tries. The error doesn't happen at https://whedon.theoj.org/, so it's hard to test.

**Comment by marcosvital** [comment]
Dear @ViniciusBRodrigues, @trashbirdecology and @petroniocandido, would you be willing to review this submission for JOSS?

We carry out an open checklist-driven reviews here in GitHub issues, and follow these guidelines: https://joss.readthedocs.io/en/latest/review_criteria.html

This is still a pre-review issue, and once there are enough reviewers we will move on to a new issue where the review will take place. If you are not able to review, please let us know if you have recommendations for other reviewers that could help with this submission.

**Comment by trashbirdecology** [comment]
I have a slight COI

**Comment by danielskatz** [comment]
If you can say a little about the COI, we can decide if it can be waived or will prevent you from reviewing.  Of course, if you prefer not to review because of the COI, that's ok too.

**Comment by ViniciusBRodrigues** [comment]
> Dear @ViniciusBRodrigues, @trashbirdecology and @petroniocandido, would you be willing to review this submission for JOSS?
> 
> We carry out an open checklist-driven reviews here in GitHub issues, and follow these guidelines: https://joss.readthedocs.io/en/latest/review_criteria.html
> 
> This is still a pre-review issue, and once there are enough reviewers we will move on to a new issue where the review will take place. If you are not able to review, please let us know if you have recommendations for other reviewers that could help with this submission.

@marcosvital and @gmyenni I think that I can help, but would be better to see the article proof first.

**Comment by gmyenni** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03027/joss.03027/10.21105.joss.03027.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03027/joss.03027/10.21105.joss.03027.pdf) :page_facing_up: :point_left:

**Comment by ethanwhite** [comment]
@ViniciusBRodrigues the proof is now linked above.

**Comment by ViniciusBRodrigues** [comment]
> Dear @ViniciusBRodrigues, @trashbirdecology and @petroniocandido, would you be willing to review this submission for JOSS?
> 
> We carry out an open checklist-driven reviews here in GitHub issues, and follow these guidelines: https://joss.readthedocs.io/en/latest/review_criteria.html
> 
> This is still a pre-review issue, and once there are enough reviewers we will move on to a new issue where the review will take place. If you are not able to review, please let us know if you have recommendations for other reviewers that could help with this submission.

Alright you can add me as reviewer.

**Comment by marcosvital** [comment]
Sorry about the long absence, @gmyenni.

@ViniciusBRodrigues, it took me a long while to be back on this submission, so I hope you can still be able to review it. I still have to find additional reviewers, so it will probably take a few days before we start.

**Comment by marcosvital** [comment]
Dear @pboesu, would you be willing to review this submission for JOSS?

We carry out an open checklist-driven reviews here in GitHub issues, and follow these guidelines: https://joss.readthedocs.io/en/latest/review_criteria.html

This is still a pre-review issue, and once there are enough reviewers we will move on to a new issue where the review will take place. If you are not able to review, please let us know if you have recommendations for other reviewers that could help with this submission.

**Comment by pboesu** [comment]
Hi @marcosvital , unfortunately I am not able to review this in the next month. Perhaps @FlukeAndFeather is interested and available?

**Comment by FlukeAndFeather** [comment]
> Hi @marcosvital , unfortunately I am not able to review this in the next month. Perhaps @FlukeAndFeather is interested and available?

@pboesu thanks for suggesting me. Yes, I would be happy to review `portalcasting`. This would be my first JOSS review - is there a sign-up process? I read through the [reviewer guidelines](https://joss.readthedocs.io/en/latest/reviewer_guidelines.html) and didn't see that mentioned. Thanks!

**Comment by ajpelu** [comment]
Dear @marcosvital, thanks for considering me to review this submission. Unfortunately, I am not able to review it. Next time.

**Comment by marcosvital** [comment]
Thank you, @FlukeAndFeather! There is no need to sign-up. The reviewing is open, and it will happen here at GitHub. A new issue will be created for that, as soon as we have enough reviewers.

 @ajpelu and @pboesu, thank you for your quick replies. :)

**Comment by marcosvital** [comment]
Dear @neerajdhanraj, would you be willing to review this submission for JOSS?

We carry out an open checklist-driven reviews here in GitHub issues, and follow these guidelines: https://joss.readthedocs.io/en/latest/review_criteria.html

This is still a pre-review issue, and once there are enough reviewers we will move on to a new issue where the review will take place. If you are not able to review, please let us know if you have recommendations for other reviewers that could help with this submission.

**Comment by marcosvital** [comment]
@whedon assign @ViniciusBRodrigues  as reviewer

**Comment by whedon** [comment]
OK, @ViniciusBRodrigues  is now a reviewer

**Comment by marcosvital** [comment]
@whedon add @FlukeAndFeather  as reviewer

**Comment by whedon** [comment]
OK, @FlukeAndFeather  is now a reviewer

**Comment by marcosvital** [comment]
Dear @jhollist, would you be willing to review this submission for JOSS?

We carry out an open checklist-driven reviews here in GitHub issues, and follow these guidelines: https://joss.readthedocs.io/en/latest/review_criteria.html

This is still a pre-review issue, and once there are enough reviewers we will move on to a new issue where the review will take place. If you are not able to review, please let us know if you have recommendations for other reviewers that could help with this submission.

**Comment by jhollist** [comment]
@marcosvital Thanks for the invite, but unfortunately I am going to have to decline.  I've got several reviews waiting on me right now and can't take on anymore for the time being.  Sorry!

**Comment by marcosvital** [comment]
No problem, @jhollist, and thank you for the quick reply.

**Comment by marcosvital** [comment]
Dear @bradyrx, would you be willing to review this submission for JOSS?

We carry out an open checklist-driven reviews here in GitHub issues, and follow these guidelines: https://joss.readthedocs.io/en/latest/review_criteria.html

This is still a pre-review issue, and once there are enough reviewers we will move on to a new issue where the review will take place. If you are not able to review, please let us know if you have recommendations for other reviewers that could help with this submission.

**Comment by marcosvital** [comment]
Dear @gmyenni, I'm sorry that it took so long for the reviewing process to start for you submission... Since that we can't find a third reviewer, I'll start the process with only two.

@ViniciusBRodrigues and @FlukeAndFeather, thank you for your patience so far. We will start the reviewing process right away.

**Comment by marcosvital** [comment]
@whedon start review

**Comment by whedon** [comment]
OK, I've started the review over in https://github.com/openjournals/joss-reviews/issues/3220.

**Comment by neerajdhanraj** [comment]
Dear Marcos,

Sorry to see this email in my Spam folder. I know it must be already too
late to respond you. Are you still awaiting for reviews on this paper?

Best Regards,

*Neeraj Dhanraj Bokde     *
Postdoctoral Researcher

Ph.D., M.E.

*     Aarhus University*

*     Department of Mechanical and Production Engineering*

     Inge Lehmanns Gade 10, Navitas

     8000  Aarhus C

     Denmark



Tel.: +45 6090 1986
E: ***@***.***

**Comment by marcosvital** [comment]
Hi, @neerajdhanraj! Thank you for letting me know, but don't worry, we already started the review.
